css/common.css

```diff
@@ -0,0 +1,75 @@
+.footer{
+	width: 100%;
+	
+	line-height: 100px;
```

Comment:
How should I handle the text in the footer to be vertically align without using <div> and without using line-height?